packages/admin-panel/src/api/mutations/useResubmitSurveyResponse.js

```diff
@@ -4,14 +4,23 @@
  *
  */
 import { useMutation } from 'react-query';
-import { post } from '../../VizBuilderApp/api/api';
+import { multipart } from '../../VizBuilderApp/api/api';
```

Comment:
Hmm, no luck :(

```
Invalid hook call. Hooks can only be called inside of the body of a function component. This could happen for one of the following reasons:
```

I think useQuery is just a global with no sense of context, whereas useApi is only exposed to react components.

Reply:
Happy with this @rohan-bes?

Reply:
@IgorNadj I tried with `useApi()` as above and it seemed to work okay for me..? You gotta do it outside of the mutation itself if that's what you did.

Reply:
Hope it's okay, I'll just push up my change since we're working async

Reply:
Hmm, not sure how it's working but great, thanks!